virtual ember conf 2021 videos & a11y testing (#944)
ember-learn/ember-blog: content/the-ember-times-issue-177.md

<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>

> Ember Conf Videos 🔥 📼,

Reviewer: ```suggestion
EmberConf Videos 🔥 📼,
```

Achieving A11y Automation Testing 🎥,

---

> ## [Virtual Ember Conf 2021 🔥 📼](https://www.youtube.com/playlist?list=PL4eq2DPpyBbkc3vz8_N4xkeboowQcPQbK)

Reviewer: ```suggestion
## [Virtual EmberConf 2021 🔥 📼](https://www.youtube.com/playlist?list=PL4eq2DPpyBbkc3vz8_N4xkeboowQcPQbK)
```

> Heyo! Did you miss out on Virtual Ember Conf 2021? Did you see some talks, but missed others? Did you watch all of the content, but want to re-watch things!? 👀 Fear not, the the suite of full conference talks can now be found online now as [individual videos](https://www.youtube.com/playlist?list=PL4eq2DPpyBbkc3vz8_N4xkeboowQcPQbK).

Reviewer: ```suggestion
Heyo! Did you miss out on Virtual EmberConf 2021? Did you see some talks, but missed others? Did you watch all of the content, but want to re-watch things!? 👀 Fear not, the the suite of full conference talks can now be found online now as [individual videos](https://www.youtube.com/playlist?list=PL4eq2DPpyBbkc3vz8_N4xkeboowQcPQbK).
```
